docs/dev.rst

```diff
@@ -5,37 +5,70 @@ Open edX development
 
 In addition to running Open edX in production, Tutor can be used for local development of Open edX. This means that it is possible to hack on Open edX without setting up a Virtual Machine. Essentially, this replaces the devstack provided by edX.
 
-The following commands assume you have previously launched a :ref:`local <local>` Open edX platform. If you have not done so already, you should run::
+.. warning::
 
-    tutor local quickstart
+  Do not run ``tutor dev ...`` commands on a production system. They can modify your configuration and database in ways that are nonsensical or insecure for production use.
```

Comment:
@regisb I felt the need to add this warning because `tutor dev quickstart` would do things you'd likely not want on a production instance:
1. import the demo course
2. create a superuser
3. automatically set `LMS_ROOT=local.overhang.io` and `ENABLE_HTTPS=false`

You've asked me to revert (1) and (2) so those aren't concerns any more.

Should I revert (3) as well? That is, should `tutor dev quickstart` prompt the user with "Is this a production instance? [y/n]" instead of assuming "no" like it does in my current changeset?

Reply:
(3) is great, let's keep it! It makes configuration much simpler and less error-prone. But doing (3) will not dramatically affect the platform, changing HTTPS/host settings is a routine task for many people. So we can get rid of the warning.

Reply:
Sounds good!